src/address.c

```diff
@@ -31,6 +32,8 @@ bool address_from_pubkey(const uint8_t public_key[static 65], uint8_t *out, size
     uint8_t address[32] = {0};
     cx_sha3_t keccak256;
 
+    LEDGER_ASSERT(out != NULL, "NULL out\n");
```

Comment:
```suggestion
    LEDGER_ASSERT(out != NULL, "NULL out");
```
I think the \n is already added when displaying the message and such?